tests/reference/cpp-complex2-cc31ff8.stdout

```diff
@@ -19,8 +19,14 @@ Kokkos::View<T*> from_std_vector(const std::vector<T> &v)
 void test_complex_cpp()
 {
     std::complex<float> c;
-    c = {423.000000, 23423.540000};
-    c = {1.000000, 2.000000};
+    c = complex(423.000000, 23423.540000);
+    c = complex(1, 2);
+}
+
+std::complex<float> complex(double x, double y)
+{
+    std::complex<float> _lpython_return_variable;
+    return _lpython_return_variable;
```

Comment:
This is not correct C++ code. We should rework the codegen for complex numbers in a separate PR.